lib/AthomApi.js

```diff
@@ -81,10 +82,10 @@ class AthomApi {
         const codePromise = new Promise(resolve => {
           app.get('/auth', (req, res) => {
             res.sendFile(path.join(__dirname, '..', 'assets', '1px.png'));
-            const { code } = req.query;
-            if (code) {
-              console.log(code);
-              resolve(code);
+            const { receivedCode } = req.query;
+            if (receivedCode) {
+              console.log(receivedCode);
+              resolve(receivedCode);
```

Comment:
```suggestion
            if (req.query.code) {
              console.log(req.query.code);
              resolve(req.query.code);
```

This is the name that appears in the url, we can't change that